src/components/Camera/Photo.js

```diff
@@ -9,8 +9,36 @@ import { screenshot } from '../utils/camera.js'
 
 import { CameraPure, CaptureActions } from './index.js'
 
-export default class Photo extends React.Component<CameraType> {
+type PhotoStateType = {
+  hasError?: boolean,
+  cameraError?: Object,
+}
+
+export default class Photo extends React.Component<CameraType, PhotoStateType> {
   webcam = null
+  selfieTimeoutId = null
+
+  state: PhotoStateType = {
+    hasError: !!this.props.hasError,
+    cameraError: this.props.cameraError,
+  }
+
+  componentDidMount () {
+    this.clearSelfieTimeout()
+    this.selfieTimeoutId = setTimeout(() => {
+      // Only show camera_inactive warning if camera hasn't errored
+      if (!this.props.hasError) {
```

Comment:
How about if instead of having a derived `hasError` prop, we had a `isCameraNotResponding` prop, along with a `cameraError`?

I find `hasError` a bit ambiguous and redundant, given we can infer there is an error if there is a `cameraError`, or the camera is not responding?